src/NzbDrone.Core/Movies/MovieRepository.cs

```diff
@@ -313,19 +313,20 @@ public List<int> AllMovieTmdbIds()
 
         public List<int> GetRecommendations()
         {
-            var recommendations = new List<List<int>>();
+            var recommendations = new List<int>();
             var tmdbIds = AllMovieTmdbIds();
```

Comment:
Blitz this, its not used anymore and just slows things down by making another call.